website/project/model.py

```diff
@@ -3879,6 +3881,13 @@ def is_rejected(self):
         else:
             return False
 
+    @property
+    def logs(self):
+        return [
+            '{} on {}'.format(v.title(), k)
```

Comment:
Try to separate presentation/formatting code from the model. This should probably be a helper function, e.g. `format_status_logs(draft)`.